### ISSUE TITLE
FlyingThings3D test set

### ISSUE THREAD

**Issue by tlkvstepan** [JiaRenChang/PSMNet#45]
Hello,

In the FlyingThing3D dataset there are several stereo pairs with a very large disparities (10^3) and negative disparities. Do you delete such images during training and testing?  Thank you in advance!

**Comment by JiaRenChang** [comment]
@tlkvstepan 
Hi, it mentioned in page 5 in the paper.
"Some pixels have large disparities and are excluded in the loss computation if the disparity is larger than the limits set in our experiment."

**Comment by tlkvstepan** [comment]
Yes but you seems to do it in test set as well, an this I think should be specially mentioned, since it directly affects benchmarking results.

**Comment by tlkvstepan** [comment]
Thanks a lot for nicely maintained code and replies!

**Comment by givasile** [comment]
@tlkvstepan @JiaRenChang So in the test set, when an image has maximum disparity > 192 you exclude it totally from your validation or you just filter out the specific pixels with a mask?

**Comment by JiaRenChang** [comment]
@givasile 
We just filtered out by a mask.

**Comment by givasile** [comment]
@JiaRenChang I have noticed that these over the limit pixels affect also the predictions on neighborhood pixels. It is common that whole big regions near those pixels tend to be predicted erroneously. If I totally exclude those images from training and evaluation, results become much better. Have you noticed this behavior?
